src/Servers/Kestrel/test/InMemory.FunctionalTests/Http2/Http2ConnectionTests.cs

```diff
@@ -1879,7 +1879,7 @@ public async Task OutputFlowControl_ConnectionAndRequestAborted_NoException()
             {
                 requestAbortedTcs.SetException(ex);
             }
-        }).DefaultTimeout();
```

Comment:
Why are we removing the timeouts?

Reply:
The real answer is because I didn't want to write a version of `.DefaultTimeout()` for `ConfiguredTaskAwaitable`. My justification is that we were not consistently applying `.DefaultTimeout()` to `InitializeConnectionAsync()` to begin with and every async thing it calls has its own timeouts.

If we were worried about it, we could just have `InitializeConnectionAsync()` always apply `.DefaultTimeout()` to its call to `InnerInitializeConnectionAsync()`, but I don't think that's necessary.

Reply:
> If we were worried about it, we could just have `InitializeConnectionAsync()` always apply `.DefaultTimeout()` to its call to `InnerInitializeConnectionAsync()`, but I don't think that's necessary.

IMO, I think we should do that. There are other HTTP/2 test helper methods that call `DefaultTimeout` internally, e.g. `ExpectFrame`

Reply:
I explained why I think it's unnecessary at https://github.com/dotnet/aspnetcore/pull/41181/files#r850645410.